config/typescript/styles.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module '*.scss' {
```

Comment:
This should already be covered by our ts config importing https://github.com/Shopify/sewing-kit/blob/master/packages/typescript-configs/definitions/styles.d.ts, do we need this too?

Reply:
I was having weird typescript issues with this today. Seemed like I needed to add it but maybe there's just something wonky going on in this branch. The CI failure seems to confirm that